geode-core/src/distributedTest/java/org/apache/geode/internal/cache/partitioned/PersistentColocatedPartitionedRegionDistributedTest.java

```diff
@@ -123,7 +123,7 @@ public class PersistentColocatedPartitionedRegionDistributedTest implements Seri
   private static volatile InternalCache cache;
   private static volatile CountDownLatch latch;
 
-  private final transient List<AsyncInvocation<Void>> asyncInvocations = new ArrayList<>();
+  private final transient List<AsyncInvocation<?>> asyncInvocations = new ArrayList<>();
```

Comment:
I don't think the changes from `AsyncInvocation<Void>` to `AsyncInvocation<?>` are necessary here.